docs/guide/file_management/file_management.md

```diff
@@ -0,0 +1,152 @@
+
+!!! note
+
+    When using the NFS [adapter](../file_management/file_management.md#adapter), configure file permissions under the `oneup_flysystem` key instead, as follows:
+
+    ``` yaml
+    oneup_flysystem:
+        adapters:
+            nfs_adapter:
+                local:
+                    permissions:
+                        file:
+                            public: 0750
+                        dir:
+                            public: 0640
+    ```
+
+### Native Flysystem handler
+
+Flysystem is a file storage library for PHP. It provides one interface to interact with many types of filesystems. 
+
+[league/flysystem](http://flysystem.thephpleague.com/) (along with [FlysystemBundle](https://github.com/1up-lab/OneupFlysystemBundle/)) is an abstract file handling library.
+
+[[= product_name =]] uses it as the default way to read and write content in form of binary files. Flysystem can use the `local` filesystem (this is the default and officially supported configuration), but is also able to read/write to `sftp`, `zip` or cloud filesystems (`azure`, `rackspace`, `S3`).
+
+#### Handler options
+
+##### Adapter
+
+The adapter is the *driver* used by Flysystem to read/write files. Adapters are declared using `oneup_flysystem`. A basic configuration might look like the following:
+
+``` yaml
+oneup_flysystem:
+    adapters:
+        default:
+            local:
+                directory: /path/to/directory
+```
+
+To learn how to configure other adapters, see the [bundle's online documentation](https://github.com/1up-lab/OneupFlysystemBundle/blob/main/doc/index.md#step3-configure-your-filesystems). 
+
+!!! note
+
+    Only the adapters are used here, not the filesystem configuration described in this documentation.
+    
+
+### DFS Cluster handler
+
+For clustering use the platform provides a custom metadata handler that stores metadata about your assets in the database. This is done as it is faster than accessing the remote NFS or S3 instance to read metadata. For further reading on setting this up, see [Clustering](../clustering.md).
+
+## Enabling BinaryFile Field indexing
+
+The indexing of all binary file Fields is disabled by default.
+To enable it, first, make sure you have installed Oracle Java/Open JDK 8 or higher and Apache Tika 1.20.
+Next, in the `config/packages` folder create a `binary_files.yaml` file with the following configuration:
+
+``` yaml
+ibexa_commerce_field_types:
+    binary_file_indexing:
+        enabled: true
+```
+
+To check what types are indexed, see under the `ibexa_commerce_search.default.index_content` parameter in `src/Siso/Bundle/SearchBundle/Resources/config/search.yml`. This parameter can be overriden, so you use it to index only specific types per SiteAccess or to extend the indexing to other file types.
```

Comment:
```suggestion
To check what types are indexed, check the `ibexa.commerce.site_access.config.search.default.index_content` Service Container parameter. This parameter can be overridden for a specific SiteAccess by replacing `default` by its name
```
not sure about the language, but this entire section is a bit incorrect. It is also not a good idea to override those parameters in that way, but since eZ Commerce didn't provide better extension point, let's keep it.